.github/CONTRIBUTING.md

```diff
@@ -61,93 +44,61 @@ To delete the development database, go to the root of the project and run `docke
 
 ## Online one-click setup for contributing
 
-Contribute to HospitalRun using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the webserver.
-
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/HospitalRun/hospitalrun-frontend)
+Open a file named `.env` and add a line `REACT_APP_HOSPITALRUN_API=http://localhost:5984` if you don't have it yet.
 
```

Comment:
This section's header is not correct. This information should go under `Start the application`

Reply:
@kumikokashii shall I do it?

Reply:
I am going to merge this PR then make the change